### PR TITLE
UI overhaul (node list)

### PR DESCRIPTION
**Pull request by BjrInt**
# Minor fixes

- Fixed a bug displaying `NaNk stakes` for compute nodes on mobile
- Remove the `package-lock.json` to avoid conflicting package managers
- Added a `dev` script to the `package.json` so that the `quasar` bin is not needed in the global path
- Removed an empty Vue file to avoid future bug

# UI Overhaul

UXfix

1. Display addresses consistently. Also copies an address when clicking on it (currently on your own wallet address in the header or in the node info modal).
2. Add an ellipse (`...`) on overflowing text using a css property. May [this (unused) helper function](https://github.com/BjrInt/aleph-account/blob/bjrint-patch-test/src/helpers/utilities.js#L6) get removed?
3. Redefine spacing/borders for a more modern look.
4. Display resource nodes so that:
 - They are more consistant with the main table display
 - You get all the information about them at first sight (without hovering)
 - All informations are available on mobile as well

### CODE REVIEW

**Comment by BjrInt** [comment]
Just a quick note regarding `CRN/CCN`and node names. 

It does not seem obvious which is the preferred way to refer to a certain node. From the table view, which I replicated into the node info modal, one could conclude the CCN/CRN is the best way to refer to a certain node. However the way info were displayed previously (a popover indicating the `CRN` when hovering a resource node's name) may indicate otherwise...

File: src/helpers/utilities.js [hunk 1]
```diff
@@ -25,3 +26,24 @@ export function ellipseText (text, maxLength = 9999) {
 export function ellipseAddress (address, width = 10) {
   return `${address.slice(0, width)}...${address.slice(-width)}`
 }
+
+export async function copyToClipboard (text) {
+  console.log('function')
```
Comment (hoh) [inline src/helpers/utilities.js]: Why keep this `console.log` ?

File: src/components/CoreNodeName.vue [hunk 2]
```diff
@@ -31,31 +30,20 @@ export default {
...
   },
   props: [
-    'node-hash',
-    'node-type'
+    'node-hash'
   ],
   data () {
     return {
-      node: null
+      node: undefined
```
Comment (hoh) [inline src/components/CoreNodeName.vue:37]: Can you explain why this switch from `null` to `undefined` ?

File: src/components/CoreNodeName.vue [hunk 3]
```diff
@@ -31,31 +30,20 @@ export default {
...
-        for (let node of this.nodes) {
-          if (node.hash === this.nodeHash) { found = node }
-        }
-      } else if (this.nodeType === 'resource') {
-        for (let node of this.resource_nodes) {
-          if (node.hash === this.nodeHash) { found = node }
-        }
-      }
-      this.node = found
+      this.node = this.nodes.find(node => node.hash === this.nodeHash)
```
Comment (hoh) [inline src/components/CoreNodeName.vue:42]: 🤩

**Review by hoh — COMMENTED** [review]
Thanks,

This looks really nice.

I reviewed every commit (see inline comments), and I see one regression and a few details to change on the commit messages:

I see an issue when displaying CRN info, the name of the "Linked to" node has disappeared:

image

Regarding the commit messages, the division looks fine but:
- The two commits with the same message should be squashed
- No merge commit should appear in the history
- They are not coherent between `[fix]` and `fix: ` style
- The nickname appears as `BonjourInternet` and `Bonjour Internet`, it is not coherent

File: src/helpers/utilities.js (hunk 1 above)
Comment (BjrInt) [reply src/helpers/utilities.js]: I was unsure about the Vue 2 call stack, needed a little refresher by trial and error. However those were deleted (see commit : 5f21ee14e6adcd61a5670b74329e636a5eac4d3b)

File: src/components/CoreNodeName.vue (hunk 2 above)
Comment (BjrInt) [reply src/components/CoreNodeName.vue:37]: Default behaviour of [Array.find](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Array/find) is to return undefined if there is no match

**Comment by BjrInt** [comment]
I rewrote the git history so it's much more cleaner and coherent. I also fixed that nasty regression

**Pull request merged**